tests/no-inheritance-anon-class.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Test fix for #659
+--DESCRIPTION--
+Unbound anon class causing segfaults, we delay copy but still cannot serialize the anon
+--FILE--
+<?php
+$task = new class extends Thread {
+    public function run()
+    {
+        $this->prop = new class {};
+		var_dump($this->prop); /* we do expect null: anon classes cannot be serialized */
```

Comment:
This feels strange... wouldn't it be better to have it throw an exception here? suddenly producing null is probably not going to be expected behaviour.